SETUP.md

```diff
@@ -107,7 +107,7 @@ $ bin/rails s # or, rails s -p 9000 (or whatever port you want to use that's not
 
 In a separate terminal:
 ```
-$ bin/webpacker-dev-server
+$ bin/webpack-dev-server
```

Comment:
> LGTM! @maebeale , IIUC this will change any other simple_forms that use checkboxes, which i think is probably a good thing. Are there any that should remain horizontal?

I checked, and it looks like the only other form that uses the check_boxes wrapper is the listing form, and I think those are being rendered through Vue currently because the interface for the tags is completely different:

![image](https://user-images.githubusercontent.com/54509406/93001863-b0e33080-f4f7-11ea-97cf-d13378757cfa.png)

Reply:
This is GREAT! Thanks @slaloggia !!! And thanks for digging in to that bulma/simple-form stuff. :)